performance/src/api_performance_tests.ts

```diff
@@ -336,23 +333,19 @@ const getloadTests = (
       getRequest(`/api/v1/roles/search/by-group/{groupId}`),
       RBAC_ENABLED,
     ),
-    test(
-      "search flat runs",
-      getRequest(
-        `/api/v1/runs?projectId=${sD?.workspace.projectId}`,
-      ),
+    test("search flat runs",
+      testRequest('POST', '/api/v1/runs', JSON.stringify({ projectId: sD?.workspace.projectId })),
       !!sD?.workspace.projectId,
     ),
     test(
       "search flat runs w/ hparam",
-      getRequest(
-        `/api/v1/runs?filter=%7B%22filterGroup%22%3A%7B%22children` +
-        `%22%3A%5B%7B%22columnName%22%3A%22hp.global_batch_size%22%2C%22kind` +
-        `%22%3A%22field%22%2C%22location%22%3A%22LOCATION_TYPE_RUN_HYPERPARAMETERS` + 
-        `%22%2C%22operator%22%3A%22%3E%22%2C%22type%22%3A%22COLUMN_TYPE_NUMBER` +
-        `%22%2C%22value%22%3A0.1%7D%5D%2C%22conjunction%22%3A%22and%22%2C%22kind` +
-        `%22%3A%22group%22%7D%2C%22showArchived%22%3Afalse%7D`,
-      ),
+      testRequest('POST', '/api/v1/runs', JSON.stringify({
```

Comment:
I think we should run the perf test once to confirm that this is working. I think it should.

Reply:
ran it -- needed to stringify the filter param. rerunning now.